src/snowflake/snowpark/session.py

```diff
@@ -469,15 +498,10 @@ def sql_simplifier_enabled(self, value: bool) -> None:
             pass
         self._sql_simplifier_enabled = value
 
-    @custom_packages_upload_enabled.setter
+    @custom_package_usage_config.setter
     @experimental_parameter(version="1.6.0")
-    def custom_packages_upload_enabled(self, value: bool) -> None:
-        self._custom_packages_upload_enabled = value
-
-    @custom_packages_force_upload_enabled.setter
-    @experimental_parameter(version="1.6.0")
-    def custom_packages_force_upload_enabled(self, value: bool) -> None:
-        self._custom_packages_force_upload_enabled = value
+    def custom_package_usage_config(self, config: Dict) -> None:
+        self._custom_package_usage_config = config
```

Comment:
I suggest we lowercase all the keys here like we do for dataframe reader/writer option setting